### DISCUSSION TITLE
Introduce yourself 👋

### DISCUSSION THREAD

**Discussion by TuringLovesDeathMetal** [lightdash/lightdash#1098] [General]
Hi! :wave: 

If you're new to the Lightdash community, welcome :blush: 

I’m Katie, one of the cofounders of Lightdash. Feel free to send me a message if there’s anything I can help you with!

Here's a little bit about me: 

I'm from Canada, but right now, I'm living out my best work-life balance in the mountains of Chamonix. 
<img src="https://user-images.githubusercontent.com/31848148/148253506-91c22358-6d75-4cb2-a1dd-891adf8feddb.png" alt="drawing" width="400"/>

I love everything product data, a [good dad joke](http://niceonedad.com), and I'm a sucker for amazing data visualizations (my favourite book is Information is Beautiful - yes, I'm that predictable).

Now it’s your turn :slight_smile:  Tell us a little bit about yourself, and what you’re into.

**Comment by pnadolny13** [comment]
Hey @TuringLovesDeathMetal and Lightdash community!

I'm Pat, and I'm a data engineer at [Meltano](https://meltano.com/) which is an open source project thats focused on integrating data tooling to work better together and to enable teams to implement DataOps best practices (i.e. version control, CICD, code reviews,etc.) across their whole data stack.

I'm helping test out new features and explore [new plugins](https://docs.meltano.com/concepts/plugins) for us to add.  I also maintain our [own Meltano implementation](https://gitlab.com/meltano/squared) that fuels our internal reporting and acts as an example projects for others to reference.

I'm really excited about Lightdash and where its headed! Theres already been some great work towards using it with Meltano and I'm hoping to keep pushing that forward to help make it even better. Right now I'm very interested in [this issue](https://github.com/lightdash/lightdash/issues/369#issuecomment-1027367930) and I'm excited to join the discussions!

ps - I'm new to these GitHub discussion so let me know if this comment should go somewhere else!

**Reply by TuringLovesDeathMetal** [reply]
Excited to have you here @pnadolny13! All of you folks at Meltano are so friendly, it's lovely 😊 

Also _very_ happy to hear that you're interested in #369 - we'll be chatting to some people next week to hear a bit about their minimum requirements and nirvana state so I might give you a shout! Would love to hear what you're thinking

**Comment by durgeshhhhhhh** [comment]
Hi everyone! 👋

I am Durgesh, a full stack developer from India with a React and Node.js background. I recently came across Lightdash and I am really impressed by what the team is building.

I want to make my first open source contribution to Lightdash. I have built and deployed joincodershub.com completely on my own using React, Redux Toolkit, Node.js, Express and MongoDB on AWS EC2 with real time chat, payment integration and email automation. It has received 35,660+ requests in 30 days with traffic from 5 countries.

Looking forward to contributing and learning from this community. Any guidance on good open-contribution issues for someone with a React and Node.js background would be really appreciated!

GitHub: github.com/durgeshhhhhhhh
Live: joincodershub.com
